### PR TITLE
Determine the nginx port at run time to generate a working link under…

### PR DESCRIPTION
**Pull request by pingers**
… non-port-80 conditions.

Also don't strip dashes and underscores from the project names per docker-compose change in 1.21 release: https://github.com/docker/compose/releases/tag/1.21.0 - See "All formats - Dashes and underscores in project names are no longer stripped out."

### CODE REVIEW

File: dsh [hunk 1]
```diff
@@ -12,9 +12,13 @@ MACHINE='default'
...
 export DOMAIN=${DOMAIN:-'test'}
 
+# Nginx proxy exposed port. Useful when using something other than 80.
+export NGINX_PORT=$(docker port nginx-proxy | awk -F ':' '{print $2}')
+
 # Used as the prefix for docker networking, container naming and nginx hostname.
-export PROJECT=$(basename ${PWD} | sed 's/[-_]//g')
+export PROJECT=$(basename ${PWD})
 
+exit 1
```
Comment (singularo) [inline dsh]: This doesn't seem right..

**Comment by pingers** [comment]
@singularo Too fast! You reviewed just before I removed it :wink:

**Comment by bart88** [comment]
nice i like 🌮 👍

**Comment by bart88** [comment]
:shipit:

**Comment by singularo** [comment]
Looks good, would be nice to have the port you want to expose by nginx as an env var. later ;-)
:shipit:

**Comment by pingers** [comment]
I actually started writing that bit and thought... nah!!!! later :smile: 

$DESIRED_NGINX_PORT=.... sounded weird though.

**Pull request merged**